core/object/script_language.cpp

```diff
@@ -513,6 +527,8 @@ void ScriptServer::save_global_classes() {
 		d["path"] = global_classes[E].path;
 		d["base"] = global_classes[E].base;
 		d["icon"] = class_icons.get(E, "");
+		d["is_abstract"] = global_classes[E].is_abstract;
```

Comment:
It would be better if `global_classes[E]` was performed only once.
```C++
const GlobalScriptClass &global_class = global_classes[E];
```

Reply:
I generally keep such optimizations out of the scope of PRs, unless it's something as pressing as the case of file system. However, it's enough that you have brought this up for me to fix it.